orbit/cmd/orbit/orbit.go

```diff
@@ -228,6 +228,7 @@ func main() {
 
 		var options []func(*osquery.Runner) error
 		options = append(options, osquery.WithDataPath(c.String("root-dir")))
+		options = append(options, osquery.WithLogPath(c.String("root-dir")))
```

Comment:
Should this be a sub-directory `$root-dir/osquery_log`?

Reply:
Sure!

Reply:
Hm, with this change, it's not working... looking into it.

Reply:
Maybe the directory needs to be created?

Reply:
No, it's created by the code already... Rolled back to this version and now I see no logs... super weird.